app/main.py

```diff
@@ -1 +1,64 @@
-# write your code here
+class SoftwareEngineer:
+    skills: list[str] = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    @classmethod
+    def learn_skill(cls, skill: str) -> None:
+        cls.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills: list[str] = ["JavaScript", "HTML", "CSS"]
```

Comment:
1) Use the `extend()` method or` +=` operator to add multiple values to a single list.
2) `skills` variable should be an attribute of the instance not class.